debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleConnectorTask.java

```diff
@@ -97,13 +97,20 @@ public ChangeEventSourceCoordinator<OraclePartition, OracleOffsetContext> start(
 
         validateRedoLogConfiguration(connectorConfig, snapshotterService);
 
-        OraclePartition partition = previousOffsets.getTheOnlyPartition();
         OracleOffsetContext previousOffset = previousOffsets.getTheOnlyOffset();
 
-        validateAndLoadSchemaHistory(connectorConfig, partition, previousOffset, schema, snapshotterService);
+        validateAndLoadSchemaHistory(connectorConfig, jdbcConnection, previousOffsets, schema, snapshotterService.getSnapshotter());
 
         taskContext = new OracleTaskContext(connectorConfig, schema);
 
+        // If the binlog position is not available it is necessary to re-execute snapshot
+        if (previousOffset == null) {
+            LOGGER.info("No previous offset found");
+        }
+        else {
+            LOGGER.info("Found previous offset {}", previousOffset);
+        }
```

Comment:
We seem to log "No previous offset found" in a few places in the Oracle connector, are all these log references necessary?  See OracleSnapshotChangeEventSource.

Reply:
Just removed the log on *SnapshotChangeEventSource